lib/route/src/Obelisk/Route.hs

```diff
@@ -825,14 +832,22 @@ _R
   -> Prism' (R f) a
 _R variant = dSumGEqPrism variant . iso runIdentity Identity
 
+{-# DEPRECATED prismEncoder "Use reviewEncoder instead" #-}
```

Comment:
As with the other deprecation:

1. Make the replacement rule blatantly clear in the deprecation warning; the user should not need to understand the deprecated function OR the new function in order to fix the deprecation warning.  (Imagine someone assigned you to bump a dep for a library you've never heard of in a codebase you're not familiar with - this is not uncommon!)
2. Move the deprecated function to a "deprecated" section at the end of the file.